package.json

```diff
@@ -3,24 +3,23 @@
   "version": "0.9.0",
   "description": "A ponyfill (doesn't overwrite the native fetch) for the Fetch specification https://fetch.spec.whatwg.org.",
   "main": "fetch-node.js",
-  "browser": "fetch-browser.js",
+  "browser": "build/fetch-browser.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "mkdirp build && browserify -t brfs -s fetchPonyfill fetch-browser.js -o build/fetch-browser.js",
+    "prepublish": "npm run build"
   },
   "author": "Mark Stanley Everitt",
   "repository": {
     "type": "git",
     "url": "git://github.com/qubyte/fetch-ponyfill.git"
   },
   "license": "MIT",
-  "dependencies": {
+  "devDependencies": {
     "brfs": "1.4.0",
+    "browserify": "13.0.0",
+    "mkdirp": "0.5.1",
     "node-fetch": "1.5.1",
```

Comment:
For `require('fetch-ponyfill')()` to work from Node, `node-fetch` needs to stay in `dependencies`.